src/resources/2020/Cartography.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  Location,
+  Monster,
+  myTurncount,
+  runChoice,
+  runCombat,
+  toUrl,
+  useSkill,
+  visitUrl,
+} from "kolmafia";
+import { have as _have } from "../../lib";
+import { get } from "../../property";
+import { $skill } from "../../template-string";
+
+export const passive = $skill`Comprehensive Cartography`;
+export const skill = $skill`Map the Monsters`;
+
+export function have(): boolean {
+  return _have(passive);
+}
+
+export function mapMonster(location: Location, monster: Monster): void {
+  useSkill($skill`Map the Monsters`);
+  if (!get("mappingMonsters")) {
+    throw new Error("Failed to setup Map the Monsters.");
+  }
+  const turns = myTurncount();
+  while (get("mappingMonsters")) {
+    if (myTurncount() > turns) {
+      throw new Error("Map the Monsters unsuccessful?");
+    }
+    visitUrl(toUrl(location));
+    runChoice(1, `heyscriptswhatsupwinkwink=${monster.id}`);
```

Comment:
I would check `handlingChoice`, and add the additional parameter to runChoice to tell it that a combat is coming rather than separately runCombatting

Reply:
The wiki says that runChoice always automates combats unless told not to